frontend/src/metabase/admin/people/containers/PeopleListingApp.jsx

```diff
@@ -1,11 +1,13 @@
 import PropTypes from "prop-types";
 import { t } from "ttag";
 import { connect } from "react-redux";
+import { Group } from "@mantine/core";
```

Comment:
For simplicity and consistency, I'd like to see this wrapped as well, even if it's just a raw export from the index file.

```suggestion
import { Group } from "metabase/ui";
```

Reply:
Yeah makes sense.